community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -0,0 +1,79 @@
+# Cross Project Council Community Fund
+
+## About The Community Fund
+
+The OpenJS Foundation’s community fund supports travel and attendance at events and activities directly related to OpenJS projects, technologies, or goals. We support travel for individuals who are:
+
+* Collaborators
+* Standards body participants
+* Speakers
+
+The fund can cover the costs, in whole or in part, of a participant's hotel, airfare, and some event registrations.
+Applicants should explore employer-sponsored funding options as their first source of financial support.
+Additionally, 25% of the community fund is allocated specifically to support diverse participants.
+Funding decisions are determined by a combination of need and potential impact.
+
+## Community Fund Rules
+
+Valid travel expenses are:
+
+* Coach airfare tickets
+* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Ground transportation to/from the airport
+
+NOTE: Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc.
+Generally, registration can be reimbursed if the attendee is speaking on behalf of OpenJS, the CPC, or OpenJS-hosted projects.
+Funding is not transferable to another person, or any other event. 
+
+Travel fund assistance can only be received once per year.
+
+## How To Apply
+
+### Submit Travel Fund Request
+
+Submit your travel fund request to the Cross Project Council by completing [this application](https://forms.gle/QDt3iqoXXB5Ycovz8).
+Please see [Deadlines and Timelines](#deadlines-and-timelines) for information about cut-off dates. 
+
+### Wait For Approval
+
+The Cross Project Council reviews requests once per month. If your request is approved, you will be notified via email.
+
+## Getting Reimbursed
+
+### Submitting Expenses
+
+After the event, kindly submit your expenses within 30 days:
+
+* Fill out these [two forms](https://drive.google.com/drive/folders/1E-dTuqnIWpZN2NP-1ioAWzK_W2zSwa_P?usp=share_link)
+* Submit the forms along with **all** receipts [here](https://form.asana.com/?k=S6lGzAjHv2uv7M8llnhO_w&d=9283783873717)
+
+Please note that expenses without receipts can not be paid.
+
+### Submitting A Trip Report
+
+The process for submitting a trip report is TBD.
+<!-- Provide a trip report by filling out the following form. Preferably, also provide pictures and video of the 
+event for use on social media. -->
+
+### Receiving Payment
+
+Payments are processed within 30 days.
```

Comment:
I'm not sure how much can be done about it but I'd like to point out that having to wait a full month for reimbursement can be a big deterrent for collaborators living in countries with a low purchase power currency that might be paying more than the equivalent of a month's of salary for an airline ticket alone.

Any improvement on this might be helpful to increasing the diversity of applicants.

Reply:
Unfortunately, this is the LF policy expense reimbursement and is not something we can change.

Reply:
I'm sorry to be annoying here, but OpenJSF is an independent organization and can have its own policy on expense reimbursement. I don't think we should block on this, but we should definitely file a separate issue and aim to address it.

Reply:
@tobie is that issue link correct? It seems to link to another issue?